src/isql/isql.epp

```diff
@@ -7209,7 +7209,7 @@ static unsigned print_item(TEXT** s, const IsqlVar* var, const unsigned length)
 				// one digit more than the specified precision when the value is 0
 				// The bug appears in TCS DSQL_DOMAIN_12 and 13
 				//
-				const double value = *var->value.asFloat;
+				const double value = *var->value.asDouble;
```

Comment:
The `value` is `float` here (see 'case SQL_FLOAT' above) so `asFloat` should be used. Use `static_cast<double>` here.